xarray/tests/test_dask.py

```diff
@@ -635,6 +635,11 @@ def counting_get(*args, **kwargs):
 
         assert count[0] == 1
 
+    def test_duplicate_dims(self):
+        data = np.random.normal(size=(4, 4))
+        arr = DataArray(data, dims=("x", "x"))
+        arr.chunk({"x": 2})
```

Comment:
1. Can we add an `assert` on `arr.chunks` which would also test that *it* works?
2. Can you also add a whats-new entry please?

Reply:
Done